parm/config/config.base.emc.dyn

```diff
@@ -227,11 +233,77 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
-# run wave component
-export DO_WAVE="NO"
+# APP settings
+export APP="ATM"
+
+if [ $APP = "ATM" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "ATMW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "S2S" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "S2SW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "ATMAERO" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="YES"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+fi
+
 if [[ "$SDATE" -lt "2019020100" ]]; then DO_WAVE="NO" ; fi # no rtofs in GDA
 export WAVE_CDUMP="both"
 
+# Coupling settings
+export cpl=".true."
+export cplflx=".true."
+if [ $DO_CICE = "YES" ]; then
+  export cplice=".true."
+else
+  export cplice=".false."
+fi
+if [ $DO_GOCART = "YES" ]; then
+  export cplgocart=".true."
+else
+  export cplgocart=".false."
+fi
+if [ $DO_WAVE = "YES" ]; then
+  export cplwav=".true."
+  export cplwav2atm=".true."
+else
+  export cplwav=".false."
+  export cplwav2atm=".false."
+fi
+
+# Temporarily override settings for aerosol coupling
+if [ $cplgocart = ".true." ]; then
```

Comment:
This block is probably no longer needed, since the switch on the APP already enforces this.

Reply:
The cplflx setting is the only one that isn't .false. and wouldn't be set to .false. if I remove that if-block. Otherwise, yes it can go. Should cplflx=.false. for now?

```# Coupling settings

export cpl=".true."
export cplflx=".true."
export cplice=".false."
export cplgocart=".false."
export cplwav=".false."
export cplwav2atm=".false."

if [ $DO_CICE = "YES" ]; then
  export cplice=".true."
fi
if [ $DO_GOCART = "YES" ]; then
  export cplgocart=".true."
fi
if [ $DO_WAVE = "YES" ]; then
  export cplwav=".true."
  export cplwav2atm=".true."
fi

# Temporarily override settings for aerosol coupling
if [ $cplgocart = ".true." ]; then
  export cplflx=".false."
  export cplice=".false."
  export cplocn=".false."
  export cplwav=".false."
  export cplwav2atm=".false."
fi
```

Reply:
cplflx should only be turned on if one of the coupling components that need it are on (MOM6/CICE, not sure about WW3)

Reply:
cplflx is for ocn/ice, wave is seperate.

Reply:
Here's what my aerosol run has for settings:

in input.nml:
cplwav       = .false.              ! CROW configured
cplchm       = .true.
cplflx       = .false.
cplwav2atm   = .false.          ! CROW configured

in model_configure:
cpl:                     .true.

Reply:
Sounds like I should add DO_OCN and this if-block?

```
export DO_OCN="NO" #default
export cplflx=".false." #default

if [ $DO_CICE = "YES" -o $DO_OCN = "YES" ]; then
  export cplflx=".true."
fi
```